pkg/sdk/resourcebuilder/component.go

```diff
@@ -425,6 +438,75 @@ func Certificate(parent reconciler.ResourceOwner, config ComponentConfig) (runti
 	return &cert, reconciler.StatePresent, nil
 }
 
+func MutatingWebhookConfiguration(parent reconciler.ResourceOwner, config ComponentConfig) (runtime.Object, reconciler.DesiredState, error) {
+	scope := admissionregistration.AllScopes
+	failurePolicy := admissionregistration.Ignore
+	sideEffects := admissionregistration.SideEffectClassNone
```

Comment:
doesn't it classify to have have a side effect?

Reply:
nvm, side effect would be something outside of the scope of the mutated resource, so this is okay